encord/objects/classification.py

```diff
@@ -52,6 +52,8 @@ def to_dict(self) -> Dict[str, Any]:
         }
         if attributes_list := attributes_to_list_dict(self.attributes):
             ret["attributes"] = attributes_list
+        else:
+            raise ValueError(f"Classification {str(self.uid)} requires attribute before use")
```

Comment:
I think this is excessively terse, TBH, but don't have a more constructive suggestion.

Reply:
I agree. I could move the validation logic higher up the stack / closer to sending over the wire and then I would feel more comfortable with
```
raise ValueError(f"Can't submit an Ontology containing a Classification without any attributes \n Classification {id_erring_classification} lacks an attribute")
``` 
or something to that effect.

Reply:
LGTM